src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -1173,9 +1173,12 @@
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\Kernel32\Interop.VerSetConditionMask.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\NtDll\Interop.NtQuerySystemInformation.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\NtDll\Interop.NtQueryInformationFile.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\User32\Interop.Constants.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\User32\Interop.LoadString.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Interop\Windows\User32\Interop.SendMessageTimeout.cs" />
+    <Compile Include="$(LibrariesProjectRoot)Common\src\Interop\Windows\User32\Interop.Constants.cs" />
```

Comment:
If you open System.Private.CoreLib.csproj in Visual Studio, are these files going to show up correctly in the project view? (ie in reasonable place of the hiearchy)

I know that we had troubles with this in the past and kept adding the `<Link>...</Link>` elements as workaround.

Reply:
VS doesn't even show me any files originating from this .projitems file that are conditionalized like `'$(TargetsWindows)' == 'true'`. It does show some files that are behind other conditions.

We need to figure out what criteria it uses. Long time ago, when I worked on this, the critera was something like -- all files in the main project, whatever their condition; no files in an imported project; reverse either case with metadata `<Visible>false</Visible>` (which could be applied by default)

I will take a look at this separately after your change, if you don't solve it already.

Reply:
> VS doesn't even show me any files originating from this .projitems fil

It does show it to me (you may need to open System.Private.CoreLib.sln):

![image](https://user-images.githubusercontent.com/6668460/71705787-e09d3e80-2d95-11ea-92eb-aa20b5537955.png)

I see that the VS has troubles with just these new files added against `$(LibrariesProjectRoot)` without `<Link>...</Link>`.